Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'bundler', '>= 1.8.4'
```

Comment:
I likely put it there for *some* reason in the original PR, but it probably doesn't matter now. I'll see when I get in.